ItemLibrary/Smartphone.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ItemLibrary
+{
+    public class Smartphone : Item
+    {
+        public string FrontCamera { get; set; }
```

Comment:
Just a question? How will it be used, when phone has few cameras. Because tendecy is that new phones have a lot of cameras

Reply:
many Shops solve this problem by grouping those cameras: front and back. and then they are describing it's properties in one place. For instace i attach an image 
![image](https://user-images.githubusercontent.com/65950287/99186541-93318200-2759-11eb-81a8-57279b0b3ed0.png)
i think we can extract this string and keep it in out FrontCamara and BackCamera fields

Reply:
What my point is, e.g., OnePlus 8 Pro has 3 end cameras, 1 wide, 1 micro and 1 more. So if we only want text, then that's fine, but if we want values, which could be compared, then we need something different. Maybe list of EndCamerasMP or else.

Reply:
Indeed it would be nice to extract those values since they may be vital in our comparing algorithm

Reply:
now understood your point, will change these fields so they will be List of ints instead of a string